sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala

```diff
@@ -42,10 +42,12 @@ trait LimitExec extends UnaryExecNode {
  * This operator will be used when a logical `Limit` operation is the final operator in an
```

Comment:
Please update this comment to mention the semantics of the `offset` as well?

Reply:
OK